README.md

```diff
@@ -168,66 +166,77 @@ bump deploy path/to/apis/ --hub my-hub --filename-pattern '*-api-{slug}-service'
 
 Simulate your API document's deployment to ensure it is valid by adding the `--dry-run` flag to the `deploy` command. It is handy in a Continuous Integration environment running a test deployment outside your main branch:
 
-```sh-session
+```shell
 bump deploy path/to/api-document.yml --dry-run --doc my-documentation --token $DOC_TOKEN
 ```
 
 Please check `bump deploy --help` for more usage details.
 
-### `bump diff [FILE]`
+### The diff command
 
-_If you want to receive automatic `bump diff` results on your Github Pull Requests you might be interested by [our Github Action](https://github.com/marketplace/actions/bump-sh-api-documentation-changelog) which also has a diff command._
-
-Please note that by default the command will always exit with a
-successful return code. If you want to use this command in a CI
-environment and want the command to fail **in case of a breaking
-change**, you will need to add the `--fail-on-breaking` flag to your
-diff command.
+Using the `diff` command can help to spot differences between the local API
+document and the latest deployed version. 
 
 #### Public API diffs
 
-From any two API documents or URLs, you can retrieve a comprehensive changelog of what has changed between them.
+From any two API documents or URLs, you can retrieve a comprehensive changelog
+of what has changed between them.
 
-```sh-session
+```shell
 $ bump diff path/to/your/file.yml path/to/your/second_file.yml
 * Comparing the two given definition files... done
 Modified: GET /consommations
   Response modified: 200
     [Breaking] Body attribute modified: energie
 ```
-> You can create as many diffs as you like without being authenticated. This is a **free and unlimited service** provided as long as you use the service fairly.
```

Comment:
any reason why we should remove this notice?

Reply:
It doesn't feel too necessary. I use loads of CLI tools which have an API behind them and I've never seen anything like this. People expect commands to work, so you dont need to let them know it will work. 

If the goal is too try and convince people to treat it gently, this will probably end up doing the opposite, so it doesnt add any value and could potentially be harmful.